src/llvm/ext/llvm-versions.txt

```diff
@@ -1 +1 @@
-11.0 10.0 9.0 8.0 7.1 6.0 5.0 4.0 3.9 3.8
+11.1 11.0 10.0 9.0 8.0 7.1 6.0 5.0 4.0 3.9 3.8
```

Comment:
In my case it's `11.1.0`

Reply:
Yeah, but it works by omitting the _patch_ part anyway - all the other versions are defined like that.